### DISCUSSION TITLE
求助，Mac M2电脑,下载zotero1.0.1版，工作区无法正常使用

### DISCUSSION THREAD

**Discussion by luckycongyu** [windingwind/zotero-better-notes#457] [Q&A] [answered]
具体问题如下：1.“我的文库”上方没有出现“打开工作区”字样；2.”我的文库“里没有“my notes”文件夹；3.文件和编辑栏目下放没有“插入模板”“使用模板插入条目”等选项。4.没有新手指引；

**Comment by windingwind** [comment, marked as answer]
是的。新版本移除了初次使用指引。打开工作区成为标签页栏的一个按钮。其他功能应当是有的

**Comment by luckycongyu** [comment]
您好，谢谢您百忙之中的回复！不过还有一个问题诶，我参考您之前做的笔记实践视频页面，和我的版本有些许不同：无法插入笔记模板，选项不太一样。具体可见附件图片，不知道是不是版本问题？期待您的回复。万分感谢！
   祝您生活愉快，科研顺利！

打开文件之后显示如下：

￼￼


点击“插入模版到工作区笔记”显示如下：
￼


￼



其他页面显示如下，供您参考了解：


￼￼

**Comment by luckycongyu** [comment]
主要是不存在“使用模板插入多条笔记”“使用模板插入多个条目”“笔记引用格式”这几个按钮，难道是需要先在“模板编辑器”中加入相应笔记和条目模板么？
